tests/int/pactGenerating.test.ts

```diff
@@ -30,9 +30,30 @@ describe('createPacts', () => {
         (queryArrayFormat) => {
             const pactsConfig = {...pactsConfigFactory('query-array-format'), commonConfigForProviders: {queryArrayFormat}};
 
-            const generatedPact = createPacts(pactsConfig)[0].pact;
+            const generatedPact = createPacts(pactsConfig)[0];
 
-            expect(JSON.parse(generatedPact)).toMatchSnapshot();
+            expect(JSON.parse(JSON.stringify(generatedPact, null, 2))).toMatchSnapshot();
         },
     );
+
+    test('throws error when pact interactions array is empty', () => {
+        const pactsConfig = {
+            consumer: 'consumer-name',
+            buildDir: 'pacts',
+            providers: [
+                {
+                    provider: 'first-provider-name',
+                    files: [`tests/int/testCases/empty-interactions/**/*.ts`],
+                },
+                {
+                    provider: 'second-provider-name',
+                    files: [`tests/int/testCases/empty-interactions/**/*.ts`],
+                },
+            ],
+        };
+
+        expect(() => createPacts(pactsConfig)).toThrowError(
+            'Pact interactions for provider: first-provider-name and consumer: consumer-name are empty.\nPact interactions for provider: second-provider-name and consumer: consumer-name are empty.',
```

Comment:
I think that the consumer name is not necessary. The consumer name is always the same, one time defined never changed, for all providers in a project. The more important is here the provider name and I think that we could keep only the provider name.

Reply:
I could see cases when there are more than one consumer - maybe monorepo-like projects, neverthless that is some distant future, right now I share the same opinion that provider will be enough.